### DISCUSSION TITLE
Accessing data from MinIO tenants across different namespace in Kubernetes

### DISCUSSION THREAD

**Discussion by Amey2400** [minio/minio#14849] [Q&A]
In Kubernetes, we cant access the same PVC in pods across different namespaces. So the data within the volume bound cant be accessed in pods across different namespaces. But if we deploy MinIO on Kubernetes and Suppose we have two namespaces ‘ns1’ and ‘ns2’ in a cluster and if the pod within ns1 has a claim requesting a specific storage volume and then the cluster dynamically provisions a volume for that PVC. So pod will access that PV and some data is stored within that PV.So PV is bound to PVC in namespace ‘ns1’. So can we access that PV within a pod which is in namespace ‘ns2’.

**Comment by elmazzun** [comment]
I am not a MinIO maintainer but as stated in [Kubernetes doc](https://kubernetes.io/docs/concepts/services-networking/network-policies/#default-policies):

_By default, if no policies exist in a namespace, then all ingress and egress traffic is allowed to and from pods in that namespace._

Did you define NetworkPolicies in MinIO Tenant Namespaces?
